docs/controllibraries.md

```diff
@@ -433,61 +431,61 @@ The code within the `render()` method is the same as in "notepad controls".
  * ${copyright}
  */
 
-sap.ui.define(['jquery.sap.global'],
-	function(jQuery) {
-	"use strict";
-
+sap.ui.define([
+],
+	() => {
```

Comment:
We had discussed the use of ES6+ in code samples earlier and decided against it - until we fully support it out of the box.

Reply:
Ahh missed to tell that - but I will take a look in more depth next week. We had discussed the change of this md already last year but missed to complete the update...

Reply:
@codeworrior done. also double checked the usage of proper api v2 renderer methods in the code i modified.